src/experimental/tools/mediaCapabilitiesProber/api/index.ts

```diff
@@ -146,8 +166,8 @@ const mediaCapabilitiesProber = {
       "matchMedia",
       "isTypeSupportedWithFeatures",
     ];
-    return probeMediaConfiguration(config, browserAPIS).then((result) => {
-      switch (result) {
+    return probeMediaConfiguration(config, browserAPIS).then(({ globalStatusNumber }) => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Update or refactor this function so that its implementation doesn't duplicate the one on line 66. ([typescript:S4144](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4144))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=315&issues=AWXy2D9D2IDoRcZ5Haqo&open=AWXy2D9D2IDoRcZ5Haqo)

[ISSUE_KEY]: AWXy2D9D2IDoRcZ5Haqo